components/global/global_header.scss

```diff
@@ -47,6 +53,43 @@
             }
         }
 
+        .channel__wrap {
+
+            // indicates the RHS is open
+            &.move--left {
+
+                // adjust channel header width
+                #channel-header {
+                    width: calc(100% - 400px);
+                }
+            }
+        }
+
+        // adjust RHS position and height
+        #sidebar-right {
+            z-index: 20;
+            top: 0;
+
+            .sidebar-right-container {
```

Comment:
nit (and probably out of scope for this ticket): for consistency-purposes this should be named `.sidebar-right__container`

Reply:
Yup. Not really in scope for this PR though as it's an existing class on an existing element. :)